src/humancellatlas/data/metadata/api.py

```diff
@@ -318,11 +318,11 @@ def __init__(self,
             if suffix == 'accessions':
                 assert prefix, name
                 assert isinstance(value, list)
-                accessions.update(Accession(domain=prefix, value=v) for v in value)
+                accessions.update(Accession(namespace=prefix, accession=v) for v in value)
         self.accessions = accessions
 
     def _accessions(self, domain: str) -> Set[str]:
-        return {a.value for a in self.accessions if a.domain == domain}
+        return {a.accession for a in self.accessions if a.namespace == domain}
```

Comment:
```suggestion
        return {a.accession for a in self.accessions if a.namespace == namespace}
```
No reason to keep the parameter named `domain`, right?

Reply:
Thanks for catching this :)